### ISSUE TITLE
Can't get empv-youtube working; no invidious API available?

### ISSUE THREAD

**Issue by k4r4b3y** [isamert/empv.el#91]
I am trying to get `empv-youtube` command working.  So, I follow empv's documentation, telling me to consult the invidious instance list here: https://api.invidious.io/

Well, the first red flag: there are only 3 instances listed there, and all of them have a red cross on their API column.  Does that mean they don't support the API functionality that `empv-youtube` needs?

Regardless, I go ahead and add the following setq line to my existing (and working) empv config:

```
(setq empv-invidious-instance "https://inv.nadeko.net/api/v1")
```

I make a search using `empv-youtube` function.  In the minibuffer, I get `Username [for {...ellided...}]:` basically asking me my username for an account on inv.nadeko.net.

----

Is it the only way left to us for using `empv-youtube` nowadays, that is, creating an account there and /then/ using the API?  I tried the other invidious instance listed in there, which simply failed in the Youtube search.

**Comment by k4r4b3y** [comment]
More: I saw an announcement on the nadeko instance: https://git.nadeko.net/Fijxu/invidious/wiki/Important-information

This is from 2 days ago, which states the following: 

> I also would like to ask Bot/API developers to not use my instance to get information about a video from Youtube as this creates more requests to Youtube therefore blocking the Backend of the instance. Please use tools like https://github.com/LuanRT/YouTube.js or any other tool that lets you get video information from Youtube.

> **API has been disabled** for everyone due to the high influx of API requests and instability of the instance.

----

Well, in light of this, what do you guys use for `empv-youtube` nowadays?

**Comment by isamert** [comment]
Yeah, unfortunately YouTube is being pretty aggressive with the blockings and all lately.

Currently, I am running my own private Invidious instance. It's basically a 2-3 user instance, so really low chance of getting blocked. You can take a look at here[1] and see if self hosting is an option for you or not.

> https://github.com/LuanRT/YouTube.js

I will check out this project and see if empv can use this instead of Invidious as the YouTube backend. If it's easy to translate the responses of the youtube.js into Invidious responses then it would be pretty easy. I'll also look around if there are other options.

[1]: https://docs.invidious.io/installation/
